development/site-components/navbar.pug

```diff
@@ -0,0 +1,26 @@
+html
```

Comment:
Pretty sure you don't need the html at the top of this file. Can you try taking it out and seeing if it still runs?